README.md

```diff
@@ -21,50 +21,66 @@ This is an app that allows a customer to view a takeaway menu and place an order
+ ```
  Add some dishes to your order
-`2.5.0 :008 > t.add_to_order("Pizza", 2)
+```
+2.5.0 :008 > t.add_to_order("Pizza", 2)
  => 16
 2.5.0 :009 > t.add_to_order("Salad", 1)
- => 22.5`
- Place your order passing in the correct total. This results in a confirmation text being sent. To set to and from phone numbers, see below section Environment Variables.
-`2.5.0 :010 > t.place_order(22.5)
- => <Twilio.Api.V2010.MessageInstance account_sid: ############## api_version: 2010-04-01 body: Sent from your Twilio trial account - Thank you! Your order was placed and will be delivered before '19:47' date_created: 2019-03-17 18:47:43 +0000 date_updated: 2019-03-17 18:47:43 +0000 date_sent:  direction: outbound-api error_code: 0 error_message:  from: +########### messaging_service_sid:  num_media: 0 num_segments: 1 price: 0.0 price_unit: USD sid: ################ status: queued subresource_uris: {"media"=>"/2010-04-01/Accounts/##############/Messages/######/Media.json"} to: +################ uri: /2010-04-01/Accounts/######/Messages/###########.json>`
+ => 22.5
+ ```
+ Place your order passing in the correct total. This results in a confirmation text being sent. To configure Twilio text message functionality, see Environment Variables section below.
+```
+2.5.0 :010 > t.place_order(22.5)
+ => <Twilio.Api.V2010.MessageInstance account_sid: ############## api_version: 2010-04-01 body: Sent from your Twilio trial account - Thank you! Your order was placed and will be delivered before '19:47' date_created: 2019-03-17 18:47:43 +0000 date_updated: 2019-03-17 18:47:43 +0000 date_sent:  direction: outbound-api error_code: 0 error_message:  from: +########### messaging_service_sid:  num_media: 0 num_segments: 1 price: 0.0 price_unit: USD sid: ################ status: queued subresource_uris: {"media"=>"/2010-04-01/Accounts/##############/Messages/######/Media.json"} to: +################ uri: /2010-04-01/Accounts/######/Messages/###########.json>
+ ```
  Place your order passing in an incorrect total. This results in an error message.
- `2.5.0 :011 > t.place_order(21.5)
+ ```
+ 2.5.0 :011 > t.place_order(21.5)
  Traceback (most recent call last):
          3: from /Users/vi1551/.rvm/rubies/ruby-2.5.0/bin/irb:11:in <main>
          2: from (irb):11
          1: from /Users/vi1551/Makers/month_1/week_2/weekend_challenge/takeaway-challenge/lib/takeaway.rb:27:in place_order
- RuntimeError (Total cost given is incorrect, please recalculate and try again)`
+ RuntimeError (Total cost given is incorrect, please recalculate and try again)
+ ```
 
  Environment Variables
  -----
  Environment variables have been used for the Twillio (sending text) configuration. To configure your own environment, follow these steps:
 1. Navigate to your project directory root and create a file called .environment
 2. Populate your .env file with the following:
-`TWILIO_ACCOUNT_SID="your twilio account sid"
+```TWILIO_ACCOUNT_SID="your twilio account sid"
```

Comment:
Good that you considered the security of environmental variables and showed potential users how to use this for their own env variables :)